debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlDatabaseSchemaTest.java

```diff
@@ -92,7 +92,7 @@ public void shouldApplyDdlStatementsAndRecover() throws InterruptedException {
         final MySqlOffsetContext offset = initializeOffset(connectorConfig);
 
         // Set up the server ...
-        offset.setBinlogStartPoint("binlog-001", 400);
```

Comment:
The original binglog filename is invalid according to the [documentation](https://dev.mysql.com/doc/refman/5.7/en/mysqld.html) and the [source code](https://github.com/mysql/mysql-server/blob/mysql-5.7.35/sql/binlog.cc#L3624-L3629):
> **mysqld** appends a numeric extension to the binary log base name to generate binary log file names